composer.json

```diff
@@ -111,7 +111,7 @@
         "symfony/dom-crawler": "~2.8 || ~3.0 || ~4.2",
         "symfony/process": "^4",
         "ymcatwincities/media_entity_document": "*",
-        "ymcatwincities/openy_activity_finder": "^3.1 || ^4.0",
+        "ymcatwincities/openy_activity_finder": "^4.0",
```

Comment:
@podarok how can we work in a situation if a new D9 project wants AF3?

Reply:
I guess this is solved in Slack https://openy.slack.com/archives/C6G4ZTLVC/p1621426840002300
4.0 releases of AF contains AFv3 UI components for backward compatibility